src/plugins/aws/config.ts

```diff
@@ -23,11 +24,19 @@ export const getAwsConfig = async (
   );
   const config = snapshot.data();
   // TODO: Support alias lookup
+  const allItems = sortBy(
+    Object.entries(config?.["iam-write"] ?? {}).filter(
+      ([, { state }]) => state === "installed"
+    ),
+    ([id]) => id
+  );
   const item = account
-    ? config?.workflows?.items.find(
-        (i) => i.state === "installed" && i.account.id === account
-      )
-    : config?.workflows?.items[0];
+    ? allItems.find(([id, { label }]) => id === account || label === account)
+    : allItems.length !== 1
```

Comment:
Can this ever be more than 1? `account` or `label` would have to match multiple items.
Suggestion: just pick the first one like we used to.

Reply:
it doesn't make any sense to just pick the first one though; I discovered this by trying to `p0 aws role ls`, which would randomly fail or pass depending on wether the first was the one that was federated login